src/targets/javascript/axios/client.ts

```diff
@@ -20,8 +20,8 @@ export const axios: Client = {
     link: 'https://github.com/axios/axios',
     description: 'Promise based HTTP client for the browser and node.js',
     extname: '.js',
-    installation: 'npm install axios --save',
   },
+  installation: () => 'npm install axios --save',
```

Comment:
can we add a test case somewhere with a function that actually uses those input args?

Reply:
added a test